src/components/OfferCard/OfferCard.js

```diff
@@ -0,0 +1,206 @@
+// @flow
+/*
+    Pillar Wallet: the personal data locker
+    Copyright (C) 2019 Stiftung Pillar Project
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+import * as React from 'react';
+import { View } from 'react-native';
+import styled from 'styled-components/native';
+import { CachedImage } from 'react-native-cached-image';
+
+// components
+import ShadowedCard from 'components/ShadowedCard';
+import { BaseText } from 'components/Typography';
+import Button from 'components/Button';
+import Spinner from 'components/Spinner';
+
+// utils
+import { fontSizes, fontStyles } from 'utils/variables';
+import { themedColors } from 'utils/themes';
+
+type ImageObject = {
+  uri: string,
+}
```

Comment:
```suggestion
};
```

Please don't forget our friends `;` on the types :smile:

Reply:
Sorry, I keep forgetting those
;;;;;;(
For some reason, we are not getting along, I suppose :D